.devops/templates/cleanup.yml

```diff
@@ -12,7 +12,7 @@ steps:
   # In theory the "workspace: clean: all" setting should handle this, but it doesn't always seem to work.
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26637&commitId=2741d456476f27da8a30ef44694dab448a3fb606&env=prod)

✅ There was 1 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.Image Example Avatar.chromium.png|0| Added|

</details>


<div id="vrtCommentFluentUI-v0"/>